cmd/admin-handlers.go

```diff
@@ -1768,3 +1769,47 @@ func (a adminAPIHandlers) KMSKeyStatusHandler(w http.ResponseWriter, r *http.Req
 	}
 	writeSuccessResponseJSON(w, resp)
 }
+
+// ServerCPUHardwareInfo holds informantion about cpu hardware
+type ServerCPUHardwareInfo struct {
+	Addr  string             `json:"addr"`
+	Error string             `json:"error,omitempty"`
```

Comment:
Error is already a string as type `error` no reason to keep as string IMO

Reply:
In one of the node is down the error returned is  custom error `NetworkError`  from rest package. Its is failing at un-marshalling , thus used string.